changelog/unreleased/enhancement-web-as-oc10-sidecar

```diff
@@ -0,0 +1,8 @@
+Enhancement: Run web as oc10 sidecar
+
+Running web inside docker was supported as application mode, in some cases it is needed to run it as a 'sidecar' deployment.
+This now is supported out of the box without any additional configuration.
+
+Copy config/config.json.sample-oc10 to config/config.json, run dev server build and start oc10 from the docker-compose file and open http://localhost:9100.
```

Comment:
Ah, I didn't pay close enough attention here. I thought that you added a third deployment in the dev docker setup. And actually I'd like that more. It would even give us the chance to run acceptance tests dockerized against app, sidecar and ocis.